schemas/common/event/rejected.example.1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@id": "https://events.adobe.io/events/id/82235bac-2b81-4e70-90b5-2bd1f04b5c7b",
+  "activitystreams:published": "2016-07-16T19:20:30+01:00",
+  "activitystreams:to": {
+    "https://ns.adobe.com/xdm-extensions/ims/user#id": "C74F69D7594880280A495D09@AdobeID",
+    "@type": "https://ns.adobe.com/xdm-extensions/ims/user"
+  },
+  "@type": "https://ns.adobe.com/xdm/common/event/rejected",
+  "xdm:objectType": "https://stock.adobe.com/content/json/2017-08-31",
```

Comment:
Is that an asset? Should it be?

Reply:
It is not an asset. It is sort of a job specification file, whose schema and definition ~lives~ will live at the URL specified by the value of the `xdm:objectType` field.